src/app/models/user.model.ts

```diff
@@ -0,0 +1,14 @@
+export interface IUser {
+    _id?: string| null;
+    firstName: string;
+    lastName: string;
+    email?: string;
+    password?: string;
+    gender: string | null;
+    dateOfBirth: string | null;
+    region: string | null;
+    city: string | null;
+    monthlySalary: string | null;
+    resume: string | null;
+    __v?: number | null;
+  }
```

Comment:
formatting here, you can configure the formatter or just always make sure you have 1 empty line at the end, so add line 15